x-pack/plugins/security_solution/server/lib/detection_engine/signals/utils.ts

```diff
@@ -384,61 +326,47 @@ export const wrapSignal = (signal: SignalHit, index: string): WrappedSignalHit =
   };
 };
 
-export const parseInterval = (intervalString: string): moment.Duration | null => {
-  try {
-    return moment.duration(parseDuration(intervalString));
-  } catch (err) {
-    return null;
-  }
+export const parseInterval = (intervalString: string): moment.Duration => {
+  return moment.duration(parseDuration(intervalString));
 };
 
 export const getDriftTolerance = ({
   from,
   to,
-  interval,
+  intervalDuration,
   now = moment(),
 }: {
   from: string;
   to: string;
-  interval: moment.Duration;
+  intervalDuration: moment.Duration;
   now?: moment.Moment;
-}): moment.Duration | null => {
+}): moment.Duration => {
   const toDate = parseScheduleDates(to) ?? now;
   const fromDate = parseScheduleDates(from) ?? dateMath.parse('now-6m');
   const timeSegment = toDate.diff(fromDate);
   const duration = moment.duration(timeSegment);
 
-  if (duration !== null) {
-    return duration.subtract(interval);
-  } else {
-    return null;
```

Comment:
I didn't see a way for duration to be null, however if there is a way I'll revert this change.